### PR TITLE
added landing page/header

### PR DESCRIPTION
**Pull request by JoyBoy92Codes**
Looks great to me!

### CODE REVIEW

**Review by JoyBoy92Codes — COMMENTED** [review]
Looks excellent to me!

File: dental-office.scss [hunk 1]
```diff
@@ -6,4 +6,25 @@ body {
     padding: 0 20px; 
 }
 
+// Index
```
Comment (JoyBoy92Codes) [inline dental-office.scss:9]: Nice work on adding comments, however I'm not sure what the '// Index' comment specifies. Maybe change it to something like // Header Section or just // Header. Something along those lines would be great. Other than that good job, keep up the great work!!!

File: index.html [hunk 2]
```diff
@@ -56,6 +56,17 @@
...
 
+  <!-- Header -->
+  <header id="header-index">
+    <div class="header-text">
+      <h1>Dental Clinic</h1>
+      <p>3221 Waialae Ave Ste 320
+        <br>Honolulu HI 96816
+      </p>
+    </div>
+    <img src="images/dentalheaderimage.svg" alt="" id="header-img">
```
Comment (JoyBoy92Codes) [inline index.html:67]: Hey Mel, be sure to add something to your alt attribute. This is important for Accessibility purposes, as well as improving our clients SEO ranking on search engine results pages. Keep up the great work!

**Review by JoyBoy92Codes — COMMENTED** [review]
A few tips....

File: index.html (hunk 2 above)
Comment (HTMLissa) [reply index.html:67]: Consider it done :) Thanks for the insightful review!

**Pull request merged**